cli/running/running_test.go

```diff
@@ -331,3 +331,20 @@ func Test_collectInstancesSingleInstanceTntCtlLayout(t *testing.T) {
 	assert.Equal(t, "", inst.ClusterConfigPath)
 	assert.Equal(t, filepath.Join(instancesEnabled, appName), inst.AppDir)
 }
+
+func Test_getInstanceName(t *testing.T) {
+	for _, tc := range []struct {
+		fullInstanceName  string
+		isClusterInstance bool
+		expected          string
+	}{
+		{"master", false, "master"},
+		{"app.master", false, "master"},
+		{"app-stateboard", false, "stateboard"},
+		{"app-master", true, "app-master"},
```

Comment:
Add a test with the same name, but `false` cluster flag. Add `app.inst-001` test.